gear/gear/database.py

```diff
@@ -1,15 +1,61 @@
 import json
 import aiomysql
+import logging
 
+log = logging.getLogger('gear.database')
 
-async def create_database_pool(autocommit=True):
+
+async def create_database_pool(autocommit=True, maxsize=10):
     with open('/sql-config/sql-config.json', 'r') as f:
         sql_config = json.loads(f.read())
-    return await aiomysql.create_pool(host=sql_config['host'],
-                                      port=sql_config['port'],
-                                      db=sql_config['db'],
-                                      user=sql_config['user'],
-                                      password=sql_config['password'],
-                                      charset='utf8',
-                                      cursorclass=aiomysql.cursors.DictCursor,
-                                      autocommit=autocommit)
+    return await aiomysql.create_pool(
+        maxsize=maxsize,
+        # connection args
+        host=sql_config['host'], user=sql_config['user'], password=sql_config['password'],
+        db=sql_config['db'], port=sql_config['port'], charset='utf8',
+        cursorclass=aiomysql.cursors.DictCursor, autocommit=autocommit)
+
+
+class Database:
+    def __init__(self):
+        self.pool = None
+
+    async def async_init(self, autocommit=True, maxsize=10):
```

Comment:
Is it another PR to unify this with the auth server and other places that use a Database?

Reply:
Yes, please.